src/Sulu/Bundle/AdminBundle/Resources/js/stores/ResourceStore/ResourceStore.js

```diff
@@ -81,8 +85,15 @@ export default class ResourceStore {
                 this.dirty = false;
             }))
             .catch(action((response: Object) => {
+                this.initialized = true;
+                this.setLoading(false);
```

Comment:
without the `ResourceTabs` forced a infinite loader and no error was shown:

https://github.com/sulu/sulu/blob/f977ffc0f480cb1d4ddee9cde0cbb8a881011d91/src/Sulu/Bundle/AdminBundle/Resources/js/views/ResourceTabs/ResourceTabs.js#L198